src/libraries/System.Formats.Asn1/pkg/System.Formats.Asn1.pkgproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Formats.Asn1.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Do we need the `uap10.0...` support? @ericstj ?

Reply:
Largely I built the things in the infrastructure commit by copying things from relevant-feeling projects and having no idea what they mean.  If this should just say netcoreapp2.0, I'll believe it (and then even understand it).  Perhaps Pkcs wasn't a good model :).

Reply:
Do we need anything other than the current netcoreapp?

Reply:
The ref project targets .NET Standard 2.0, so he NEEDS to have all of those or else validation will fail. Basically what this supported framework list means, is that you expect that this reference project should support all of those frameworks. If he removes one, like uap, then validation will fail with something like: "uap version bla bla shouldn't be supported but it has compile and runtime assets".